packages/commands/src/evmDeposit.ts

```
@@ -0,0 +1,125 @@
+      throw new Error(
+        `Failed to create signer from private key: ${error.message}`
+      );
+    }
+    const client = new ZetaChainClient({ network: networkType, signer });
+    const tx = await client.evmDeposit({
+      amount: options.amount,
+      erc20: options.erc20,
+      receiver: options.receiver,
+      revertOptions: {
+        revertAddress: options.revertAddress,
+        callOnRevert: options.callOnRevert,
```

ESLint

[error] Expected object keys to be in ascending order. 'callOnRevert' should be before 'revertAddress'.

(sort-keys-fix/sort-keys-fix)